Makefile

```diff
@@ -1,4 +1,5 @@
-KDIR ?= /lib/modules/${kernelver}/build
+KVERSION ?= `uname -r`
+KDIR ?= /lib/modules/${KVERSION}/build
```

Comment:
I think using ":=" would be more appropiate for these assignments to avoid environment variables overriding them. Command line variables will still take precedence.